Makefile

```diff
@@ -114,7 +114,7 @@ tofnd-client:
 ###                                Protobuf                                 ###
 ###############################################################################
 
-proto-all: proto-update-deps proto-format proto-lint proto-gen
+proto-all: proto-format proto-lint proto-gen
```

Comment:
remove for now?

Reply:
I rolled the change back. We should talk about the dependency updates sometime this week